secretmanager/secretmanager_test.go

```diff
@@ -327,6 +328,23 @@ func TestCreateSecretWithLabels(t *testing.T) {
 
 	if got, want := b.String(), "Created secret with labels:"; !strings.Contains(got, want) {
 		t.Errorf("createSecretWithLabels: expected %q to contain %q", got, want)
+  }
+}
+func TestCreateRegionalSecret(t *testing.T) {
+	tc := testutil.SystemTest(t)
+
+	secretID := "createRegionalSecret"
+	locationID := testLocation(t)
+
+	defer testCleanupRegionalSecret(t, fmt.Sprintf("projects/%s/locations/%s/secrets/%s", tc.ProjectID, locationID, secretID))
```

Comment:
Hi @vipul7499 - I'd like to ask a favor of you because our goal with pull requests includes leaving code better than we found it. There are a number of tests in this file that have a `defer` statement preceding the actual resource acquisition. Would you mind not only updating your test but also fixing any of the other tests to help enhance consistency? Move `defer` statements so that they immediately follow the resource acquisition statement (in this specific case, line 346). Thank you, good citizen! :)

Reply:
Fixed.